apps/api/src/auth/service/auth.service.ts

```diff
@@ -131,6 +131,24 @@ export class AuthService {
     }
   }
 
+  /* istanbul ignore next */
+  async handleGoogleOAuth(
```

Comment:
I feel that the `handleGithubOAuth` and `handleGoogleOAuth` does the same thing. Can you merge them into a single function? say `handleOAuthLogin`?

Reply:
> I feel that the `handleGithubOAuth` and `handleGoogleOAuth` does the same thing. Can you merge them into a single function? say `handleOAuthLogin`?

Is this alright?

Reply:
Yep! This looks good. I will merge it after I generate the tokens.